### PR TITLE
Modified version

### PR DESCRIPTION
**Pull request by kjinlokm**
The modified version, it
        -Store the uploaded file in file system;
        -Download file from file system;

To run the application(if you are using Gradle), please:
        $./gradlew bootRun

or build the a JAR first, then run it:
        $./gradlew build
        $java -jar build/libs/gs-uploading-files-0.1.0.jar

With the server running, you need to open a browser and visit
'http://localhost:8080/upload' to see the upload form.

### CODE REVIEW

File: complete/src/main/java/hello/servletContext.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package hello;
+
+import java.util.*;
+
+import javax.servlet.ServletContext;
+
+public class servletContext  {
```
Comment (isobretatel) [inline complete/src/main/java/hello/servletContext.java]: Class names should be nouns, in mixed case with the first letter of each internal word capitalized. 
http://www.oracle.com/technetwork/java/codeconventions-135099.html

Comment (isobretatel) [inline complete/src/main/java/hello/servletContext.java]: servletContext referencing ServletContext is confusing.

File: complete/src/main/java/hello/servletContext.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	private ServletContext servletContext;
+	private String absoluteDiskPath;
```
Comment (isobretatel) [inline complete/src/main/java/hello/servletContext.java]: String has no validation. It there a better type to represent file path?

File: complete/src/main/java/hello/findFile.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+				if (fil.isDirectory()){
+					findFile(name,fil);
+				} else if (name.equalsIgnoreCase(fil.getName())){
+					return true;
+				}
+			}
+		}
+		return false;
+	} else {
+		System.out.println("FILE NOT ILLEGAL, STOP!");
```
Comment (isobretatel) [inline complete/src/main/java/hello/findFile.java]: http://stackoverflow.com/questions/8601831/do-not-use-system-out-println-in-server-side-code

Comment (isobretatel) [inline complete/src/main/java/hello/findFile.java]: Throw an exception instead.

**Comment by dingj2012** [comment]
Update commited: 1.DiskPath type changed to file; 2.Exception added

**Pull request closed without merging**